yt_dlp/extractor/arte.py

```diff
@@ -59,33 +46,21 @@ class ArteTVIE(ArteTVBaseIE):
         'url': 'https://www.arte.tv/de/videos/110203-006-A/zaz/',
         'only_matching': True,
     }, {
-        'note': 'age-restricted',
-        'url': 'https://www.arte.tv/de/videos/006785-000-A/the-element-of-crime/',
-        'info_dict': {
-            'id': '006785-000-A',
-            'description': 'md5:c2f94fdfefc8a280e4dab68ab96ab0ba',
-            'title': 'The Element of Crime',
-            'timestamp': 1696111200,
-            'duration': 5849,
-            'thumbnail': 'https://api-cdn.arte.tv/img/v2/image/q82dTTfyuCXupPsGxXsd7B/940x530',
-            'upload_date': '20230930',
-            'ext': 'mp4',
-        },
-    }, {
-        'url': 'https://www.arte.tv/de/videos/085374-003-A/im-hohen-norden-geboren/',
+        'url': 'https://www.arte.tv/fr/videos/109067-000-A/la-loi-de-teheran/',
         'info_dict': {
-            'id': '085374-003-A',
+            'id': '109067-000-A',
             'ext': 'mp4',
-            'description': 'md5:ab79ec7cc472a93164415b4e4916abf9',
-            'timestamp': 1702872000,
-            'thumbnail': 'https://api-cdn.arte.tv/img/v2/image/TnyHBfPxv3v2GEY3suXGZP/940x530',
-            'duration': 2594,
-            'title': 'Die kurze Zeit der Jugend',
-            'alt_title': 'Im hohen Norden geboren',
-            'upload_date': '20231218',
+            'description': 'md5:d2ca367b8ecee028dddaa8bd1aebc739',
+            'timestamp': 1713927600,
+            'thumbnail': 'https://api-cdn.arte.tv/img/v2/image/3rR6PLzfbigSkkeHtkCZNF/940x530',
+            'duration': 7599,
+            'title': 'La loi de Téhéran',
+            'alt_title': None,
```

Comment:
If the correct expected value is `None`, it doesn't need to be in the test. If not, put the correct value here and let the test fail
```suggestion
```